s1pop.css

```diff
@@ -0,0 +1,415 @@
+body{
+    font-family: 'Raleway', sans-serif ;
+    height:100%;
+    box-sizing:border-box;
+
+}
+.navbar{
+    top:0%;
+    height:10%;
+    position:relative;
+}
+ul{
+
+    display:flex;
+    flex-direction:row;
+    flex-wrap:wrap;
+    justify-content: flex-end;
+    align-items: center;
+    font-size:18px;
+}
+ul li{
+    list-style-type:none;
+    padding:0 5% 0 5%;
+    transition:all ease-in 0.5;
+}
+ul li:hover{
+    font-size:1.3em;
+}
+.navbar i{
+    position:absolute;
+    left:2%;
+    top:2%;
+    
+}
+.pojemnik {
+    position: relative;
+    height:50%;
+    width:100%;
+    margin-top:2%;
+    text-align: center;
+
+}
+.pojemnik img{
+    position:relative;
+    width: 100%;
+    height: 25em;
+    opacity:0.9;
+}
+.pojemnik .Napis{
+    text-align:center;
+    position:absolute;
+    font-size:2.5em;
+    font-weight:bold ;
+    top:10%;
+    color:#FFC53D;
+    left:35%;
+    z-index:1;
+}
+.pojemnik .Napis p{
+
+    font-size:0.7em;
+    position:absolute;
+    top:55%;
+    left:10%;
+    letter-spacing: 0.02em;
+    color:white;
+
+}
+.Pz{
+   
+    text-align:center;
+    font-style:normal;
+    margin-top:2%;
+    font-size:2.3em;
+    letter-spacing: 0.02em;
+}
+.Pz button{
+margin: 2% 2% 0 2%;
+font-size:0.8em;
+width: 20%;
+height: 20%;
+border-radius: 15%;
+background: linear-gradient(270deg, #FFC53D 14%, #FF9A3D 68.5%);
+transition:all ease-in-out 1;
+}
+.Pz button:hover{
+    background: white;
+    border-color: red;
+    color:red;
+}
+.Aboutme{
+    margin: 5% 5% 5% 5%;
+    display:flex;
+    flex-direction:row;
+    flex-wrap:wrap;
+    align-items: center;
+    justify-items: center;
+    justify-content:center;
+    
+}
+
+/* .Aboutme .p1{
+    width: 50%;
+    margin-right:2%;
+    font-weight: bold;;
+    text-align: right;
+    display:flex;
+    flex-direction:row;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    font-size:1.2em;
```

Comment:
Dobra praktyka! Nie zostawiajmy zakomentowanego kodu. Jeżeli coś jest zakomentowane to zapewne nie jest używane :) a jak nie używane to nie potrzebne :)